src/radical/pilot/configs/resource_ornl.json

```diff
@@ -316,10 +310,10 @@
         "lrms"                        : "LSF_SUMMIT",
         "lfs_per_node"                : "/tmp",
         "agent_type"                  : "multicore",
-        "agent_config"                : "default",
-        "agent_scheduler"             : "CONTINUOUS",
+        "agent_config"                : "summit_sa",
+        "agent_scheduler"             : "HOMBRE",
```

Comment:
Not sure we want this scheduler as the default one.

Reply:
ah, thanks - fixed.